tests/test_bs.py

```diff
@@ -332,7 +346,7 @@ def save_log_on_exit(request):
 
 
 @pytest.fixture(scope='module')
-def add_curl(request, setup_bootstrap):
+def add_curl(request, init_session, setup_bootstrap):
```

Comment:
maybe there's a more elegant way to make sure init_session happened instead of including it in every fixture and test ?

Reply:
ok.

Reply:
I didn't understand your answer.

Reply:
init_session, need to be called in many scenarios. This fixture composes few smaller fixtures that need to be called in every test.
I will look into how to do it more elegant (But this is not the scope of this PR)